### PR TITLE
Replace AddressBook instances with SocialBook

### LINKED ISSUE #95: Replace address book instances in docs/

### PR DESCRIPTION
**Pull request by matchastack**
closes #95

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -180,19 +180,17 @@ Format: `exit`
 
 ### Saving the data
 
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
+SocialBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
 
 ### Editing the data file
 
-AddressBook data are saved automatically as a JSON file `[JAR file location]/data/socialbook.json`. Advanced users are welcome to update data directly by editing that data file.
+SocialBook data are saved automatically as a JSON file `[JAR file location]/data/socialbook.json`. Advanced users are welcome to update data directly by editing that data file.
```
Comment (Andrew22Teoh) [inline docs/UserGuide.md:187]: The implementation for this hasn't been changed yet, it's still an addressbook.json. Just to take note of, might be fixable in time.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -180,19 +180,17 @@ Format: `exit`
...
 
 <box type="warning" seamless>
 
 **Caution:**
 If a person's data values are changed to an invalid format, Socialbook will discard that particular person's data while keeping the rest. However, if your changes to the data file makes the file format invalid, SocialBook will discard all data and start with an empty data file at the next run. Hence, it is recommended to take a backup of the file before editing it.<br>
```
Comment (Andrew22Teoh) [inline docs/UserGuide.md:192]: First "Socialbook" in this line, should be "SocialBook" for consistency.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -35,7 +35,7 @@ Given below is a quick overview of main components and how they interact with ea
```
Comment (Andrew22Teoh) [inline docs/DeveloperGuide.md:35]: Main header for Developer Guide is still "AB-3 Developer Guide", should we change it to "SB-3"?

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -84,7 +84,7 @@ The `UI` component,
 
 ### Logic component
```
Comment (Andrew22Teoh) [inline docs/DeveloperGuide.md:85]: Line 104 under the logic component references "AddressBookParser", might be good to change it if we're changing the class name too. Same in Line 115.

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -116,14 +116,14 @@ How the parsing works:
 * All `XYZCommandParser` classes (e.g., `AddCommandParser`, `DeleteCommandParser`, ...) inherit from the `Parser` interface so that they can be treated similarly where possible e.g, during testing.
 
 ### Model component
```
Comment (Andrew22Teoh) [inline docs/DeveloperGuide.md:118]: Line 133 makes mention of AddressBook, might be worth changing too?

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -139,12 +139,12 @@ The `Model` component,
...
 
-**API** : [`Storage.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/storage/Storage.java)
+**API** : [`Storage.java`](https://github.com/AY2425S1-CS2103-F10-2/tp/blob/master/src/main/java/seedu/address/storage/Storage.java)
 
 <puml src="diagrams/StorageClassDiagram.puml" width="550" />
 
 The `Storage` component,
-* can save both address book data and user preference data in JSON format, and read them back into corresponding objects.
+* can save both social book data and user preference data in JSON format, and read them back into corresponding objects.
 * inherits from both `AddressBookStorage` and `UserPrefStorage`, which means it can be treated as either one (if only the functionality of only one is needed).
```
Comment (Andrew22Teoh) [inline docs/DeveloperGuide.md]: Similar problem as earlier, "AddressBookStorage" should be changed if the class name changes.

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -164,40 +164,40 @@ This section describes some noteworthy details on how certain features are imple
```
Comment (Andrew22Teoh) [inline docs/DeveloperGuide.md]: Many references to ____AddressBook___ within this section, do you think it's worth changing?

**Review by Andrew22Teoh — CHANGES_REQUESTED** [review]
Generally looks good, main changes are just Developer Guide header "AB-3" and the model component line 133 in DeveloperGuide.md.

Everything else is regarding the class names if we do change them, can be ignored if we're not.

**Comment by matchastack** [comment]
Will not be changing the class names for now.

**Review by Andrew22Teoh — APPROVED** [review]
LGTM!

**Pull request merged**